Makefile

```diff
@@ -224,8 +224,11 @@ endef
 install: whoami
 	@helm dep up ./helm/cas-ggircs
 	@helm upgrade --install --atomic --timeout 900s \
-		--namespace $(GGIRCS_NAMESPACE_PREFIX)-$(ENVIRONMENT) --set image.etl.tag=$(GIT_SHA1) \
-		--set image.ecccUpload.tag=$(GIT_SHA1) --set image.ecccExtract.tag=$(GIT_SHA1) \
+		--namespace $(GGIRCS_NAMESPACE_PREFIX)-$(ENVIRONMENT) \
+		--set image.etl.tag=$(GIT_SHA1) \
+		--set image.ecccUpload.tag=$(GIT_SHA1) \
+		--set image.ecccExtract.tag=$(GIT_SHA1) \
+		--set swrsGcpApi.image.tag=$(GIT_SHA1) \
```

Comment:
This parameter might need to be added to the `test-local-cluster` command as well, as it is currently failing to deploy